### DISCUSSION TITLE
Only push accepted translations in the git repository

### DISCUSSION THREAD

**Discussion by kolaente** [WeblateOrg/weblate#6257] [Q&A]
Hey there!

In my application I'd like to only include strings in the connected git repo that were approved by someone, either peer reviewed or by a dedicated reviewer.
Is there a way to achieve this when using one of the [review workflows](https://docs.weblate.org/en/latest/workflows.html#dedicated-reviewers)? It currently looks like all translation strings are stored directly in the git repo once saved with no clear indication in the repo if it was approved or not.

I'm not sure if that's a configuration issue on my end or a feature request.

**Comment by nijel** [comment]
Currently, everything gets committed to the repository. Even with review workflow. It properly marks state of the strings for formats which support this (XLIFF, see https://docs.weblate.org/en/latest/formats.html#translation-types-capabilities).

See https://github.com/WeblateOrg/weblate/issues/3745 for feature request for this.

**Reply by kolaente** [reply]
I see, thanks for the related links. I'm using nested json files so using XLIFF won't work for me.

**Reply by nijel** [reply]
See also https://github.com/WeblateOrg/weblate/issues/3436
